doc/howto/cluster_recover.md

```diff
@@ -0,0 +1,120 @@
+
+To do so, complete the following steps:
+
+1. Log on to any surviving member of your cluster and run the following command:
+
+       sudo lxd cluster list-database
+
+   This command shows which cluster members are configured as database nodes.
+1. Pick one of the listed database nodes that is still online and log on to the machine (if it differs from the one you are already logged on to).
+1. Make sure that the LXD daemon is not running on the machine.
+   For example, if you're using the snap:
+
+       sudo snap stop lxd
+
+1. Run the following command:
+
+       sudo lxd cluster recover-from-quorum-loss
+
+1. Start the LXD daemon again.
+   For example, if you're using the snap:
+
+       sudo snap start lxd
+
+The database should now be back online.
+No information has been deleted from the database.
+All information about the cluster members that you have lost is still there, including the metadata about their instances.
+This can help you with further recovery steps if you need to re-create the lost instances.
+
+To permanently delete the cluster members that you have lost, force-remove them.
+See {ref}`cluster-manage-delete-members`.
+
+## Recover cluster members with changed addresses
+
+If some members of your cluster are no longer reachable, or if the cluster itself is unreachable due to a change in IP address or listening port number, you can reconfigure the cluster.
+
+To do so, edit the cluster configuration on each member of the cluster and change the IP addresses or listening port numbers as required.
+You cannot remove any members during this process.
+
+You can edit the {ref}`clustering-member-roles` of the different members, but with the following limitations:
+
+- A cluster member that does not have a `database*` role cannot become a voter, because it might lack a global database.
+- At least two members must remain voters (except in the case of a two-member cluster, where one voter suffices), or there will be no quorum.
+
+Log on to each cluster member and complete the following steps:
+
+1. Stop the LXD daemon.
+   For example, if you're using the snap:
+
+       sudo snap stop lxd
+
+1. Run the following command:
+
+       sudo lxd cluster edit
+
+1. Edit the YAML representation of the information that this cluster member has about the rest of the cluster:
+
+   ```yaml
+   # Latest dqlite segment ID: 1234
+
+   members:
+     - id: 1             # Internal ID of the member (Read-only)
+       name: node1       # Name of the cluster member (Read-only)
+       address: 192.0.2.10:8443 # Last known address of the member (Writeable)
+       role: voter              # Last known role of the member (Writeable)
+     - id: 2
+       name: node2
+       address: 192.0.2.11:8443
+       role: stand-by
+     - id: 3
+       name: node3
+       address: 192.0.2.12:8443
+       role: spare
+   ```
+
+   You can edit the addresses and the roles.
+
+1. Start the LXD daemon again.
```

Comment:
Is it important when I do the restart?
Can I log onto the first server, do the change and restart, then onto the second, change and restart and so on?
Or do I stop LXD everywhere, do the changes everywhere, and then restart everywhere?

Reply:
@ru-fu I suspect the latter option of stop everywhere, make changes and then restart everywhere, but @MathieuBordere might be able to advise further.

Reply:
It should be stop everywhere, change everywhere and then restart everywhere.